concepticondata/conceptlists/Alpher-1999-151.tsv

```diff
@@ -41,7 +41,7 @@ Alpher-1999-151-39	1256	HEAD	39	head
 Alpher-1999-151-40	2648	HAIR (HEAD)	40	head hair
 Alpher-1999-151-41	1408	HEAR	41	hear
 Alpher-1999-151-42	1223	HEART	42	heart
-Alpher-1999-151-43	353	STRIKE	43	to hit (with hand)
+Alpher-1999-151-43	1433	HIT	43	to hit (with hand)
```

Comment:
If the difference between HIT and STRIKE is one of intentionality (the former being ambiguous, the latter being intentional), then I would assume that this elicitation gloss makes the intentionality clear enough.